### DISCUSSION TITLE
Filling empty space

### DISCUSSION THREAD

**Discussion by Scouter407** [jetelain/ArmaRealMap#98] [Q&A] [answered]
maps generate with a LOT of empty space, especially around houses and places that aren't technically forests but should have trees, is there any way to change this besides going in and manually placing every single tree in 3den? I'm not familiar with the terrain editor so sorry if this is a dumb question.

**Comment by jetelain** [comment, marked as answer]
Hi,
Some feature are planned to fill empty area with some random objects, but it's not yet available : #43 .
Please note that some areas are not yet interpreted / filled, it will change with upcoming versions.
Results will greatly depends on OpenStreetMap data quality. You may contribute to OpenStreetMap to add more data useful for the generator.

**Reply by Scouter407** [reply]
Thank you, and yeah that will be great to see, as this tool is amazing <3
